<ui version="4.0" >
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelLanguage" >
     <property name="text" >
      <string>Application language:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="comboBoxLanguages" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QCheckBox" name="checkBoxCheckForUpdate" >
     <property name="text" >
      <string>Check for CeB update at startup</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QCheckBox" name="checkBoxAlwaysOnTop" >
     <property name="text" >
      <string>Keep CeB above other windows</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>203</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
